cs41_final.sh

```diff
@@ -1,11 +1,10 @@
 #!/bin/bash
 
-
 ###########################
 ###FUNCTION DECLARATIONS###
 ###########################
 
-function show_banner{
+function show_banner{ 
```

Comment:
this isn't valid.  see http://mywiki.wooledge.org/BashGuide/CompoundCommands#Functions for some examples. 

That site is a gold mine: http://mywiki.wooledge.org/BashGuide/